build.sh

```diff
@@ -1,49 +1,62 @@
-#!/bin/bash
+#! /usr/bin/env bash
 
 set -e # exit if a command fails
 set -o pipefail # Will return the exit status of make if it fails
 
-CFLAGS_COMMON="-O0 -g -DXBOX=1" # Compilation flags for all platforms
-POST_BUILD=""
-
 package_windows() { # Script to prepare the windows exe
-    mkdir -p dist
-    cp i386-softmmu/qemu-system-i386.exe dist/xqemu.exe
-    cp i386-softmmu/qemu-system-i386w.exe dist/xqemuw.exe
-    python2 ./get_deps.py dist/xqemu.exe dist
-    strip dist/xqemu.exe
-    strip dist/xqemuw.exe
+	mkdir -p dist
+	cp i386-softmmu/qemu-system-i386.exe dist/xqemu.exe
+	cp i386-softmmu/qemu-system-i386w.exe dist/xqemuw.exe
+	python2 ./get_deps.py dist/xqemu.exe dist
+	strip dist/xqemu.exe
+	strip dist/xqemuw.exe
 }
 
+postbuild=''
+build_cflags='-O3'
+job_count='4'
+
+if [ "${1:0:2}" = '-j' ]
+then
+	job_count="${1:2}"
+	shift
+fi
+
+if echo " ${@} " | grep -q '[[:space:]]--enable-debug[[:space:]]'
```

Comment:
Either we go back to the previous style #115 of looping, or this should be in a function which hints what this does:

```
function find_in_arguments() {
  echo " ${@} " | grep -q '[[:space:]]$1[[:space:]]'
}
...
if $(find_in_arguments("--enable-debug")); then
```

*(Don't trust my bash code - it's untested)*